### ISSUE TITLE
WARN deprecated @vitejs/plugin-react-refresh

### ISSUE THREAD

**Issue by folt** [JS-banana/vite-react-ts-antd-eslint-prettier#3]
Hi!
Thanks for the awesome template. I am inspired by him in my project.
I noticed a message after installation
`npm WARN deprecated @vitejs/plugin-react-refresh@1.3.6: This package has been deprecated in favor of @vitejs/plugin-react`  
Maybe update the dependencies?

**Comment by JS-banana** [comment]
> Hi! Thanks for the awesome template. I am inspired by him in my project. I noticed a message after installation `npm WARN deprecated @vitejs/plugin-react-refresh@1.3.6: This package has been deprecated in favor of @vitejs/plugin-react` Maybe update the dependencies?

Thanks for the feedback, it doesn't affect current usage, perhaps you could use pnpm to install the dependencies

**Comment by JS-banana** [comment]
> Hi! Thanks for the awesome template. I am inspired by him in my project. I noticed a message after installation `npm WARN deprecated @vitejs/plugin-react-refresh@1.3.6: This package has been deprecated in favor of @vitejs/plugin-react` Maybe update the dependencies?

Thank you for your feedback, based on the issue you described I was able to identify it in the official instructions (https://www.npmjs.com/package/@vitejs/plugin-react-refresh) and I have now completed the dependency @vitejs/plugin-react-refresh to @ vitejs/plugin-react change, thanks again for the suggestion.

And I also found that react-router has been updated to version 6.x. The current way of writing may not be elegant, I also plan to refactor the routing module in the next, welcome to participate together~

Thanks!
